pages/privacy.md

```diff
@@ -4,21 +4,113 @@ permalink: /privacy/
+
+    &nbsp;&nbsp;&nbsp;&nbsp;5.1.6. to enable us to contact you or communicate with you on any matters relating to your access or use of the Service, including but not limited to the purposes set out above, via email, push notifications or such other forms of communication that we may introduce from time to time depending on the functionality of the Service and/or your device.
+
+
+    5.2. We may share necessary data with other Public Sector Entities, and third party service providers in connection with the Service, so as to provide the Service to you in the most efficient and effective way unless such sharing is prohibited by law.
+
+
+    5.3. We will NOT share your personal data with entities which are not Public Sector Entities, except where such sharing is necessary for such entities to assist us in providing the Service to you or for fulfilling any of the purposes herein.
+
+    5.4. For your convenience, we may also display to you data you had previously supplied us or other Public Sector Entities. This will speed up the transaction and save you the trouble of repeating previous submissions. Should the data be out-of-date, please supply us the latest data.
+
+    5.5. Please note that we may be required to disclose your data by law, including any law governing the use/provision of any service of a third party service provider or collaborator.
+
+6. You may withdraw your consent to the use and disclosure of your data by us with reasonable notice and subject to any prevailing legal or contractual restrictions; however, doing so may prevent the proper functioning of the Service and may also result in the cessation of the Service to you.
+
+    **Protection of data**
+
+7. To safeguard your personal data, all electronic storage and transmission of personal data is secured with appropriate security technologies.
+
+8. The Service may contain links to external sites whose data protection and privacy practices may differ from ours. We are not responsible for the content and privacy practices of these other websites and encourage you to consult the privacy notices of those sites.
+
+    **Contact information**
+
+9. Please contact [sgid@open.gov.sg](mailto:sgid@open.gov.sg) if you:
+
+    9.1. have any enquiries or feedback on our data protection policies and procedures; or
+		
+    9.2. need more information on or access to data which you have provided to us in the past.
+
+    **Definitions**
+10. In this Privacy Statement, “Public Sector Entities” means the Government (including its ministries, departments and organs of state) and public authorities (such as statutory boards) and “personal data” shall have the same meaning as its definition in the Personal Data Protection Act 2012 (No. 26 of 2012).
+
+This version of the Privacy Statement is dated 28/11/2022.
+
+### Annex
+1. **Name of Service**: sgID mobile
+2. **Types of data requested/collected**
```

Comment:
We are also getting `HDB Type` and `Housing Type` as of a couple weeks ago

Reply:
updated!